src/flavors/DQMC/DQMC.jl

```diff
@@ -37,8 +37,7 @@ function DQMCParameters(;global_moves::Bool = false,
                         measure_rate::Int   = 10,
                         kwargs...)
     nt = (;kwargs...)
-    keys(nt) == (:beta)
-    nt = (;beta=nt.beta, delta_tau=0.1)
+    keys(nt) == (:beta) && nt = (;beta=nt.beta, delta_tau=0.1)
```

Comment:
The assignment needs to be in paratheses.

Reply:
Done